packages/server/src/parse-chokidar-rules-from-gitignore.ts

```diff
@@ -1,6 +1,6 @@
 import parseGitignore from 'parse-gitignore'
 import fs from 'fs'
-import _ from 'lodash'
+import {partition} from 'lodash'
```

Comment:
this will only reduce bundle size when using [babel-plugin-lodash](https://github.com/lodash/babel-plugin-lodash) or `lodash/es`

you could use `import partition from 'lodash/partition` which does not require tree-shake and take advantage of `lodash` "per file" repo strucure

Reply:
Thanks for the input, I'll change it to `lodash/partition`. I'll open an issue on tree shaking anyway, since I don't feel like this is the kind of issues contributors should have to worry about.

Reply:
Welp, now that I think about it - do we have to care about bundle size? I don't think Lodash will be bundled into our code, anyway, since this won't be run client-side, does it?

Reply:
You are right we shouldn't be too concerned about this instance and I am not as it is there as a transient dependency anyway and we don't control it in a bunch of stuff. I am more worried about precedent honestly especially if it is used in the frontend as this is the first time our code uses it. 

I don't think it is a big deal but honestly the best practice here is probably to use a deep import: 

```
import partition from 'lodash/partition';
```

That way if that was copied everywhere in the codebase we might be able to save size in the packages we could.

Reply:
I don't think we need an issue on tree shaking btw.